README.md

```diff
@@ -527,11 +540,10 @@ cmake --install . --prefix <path_to_install_dir>
 ```bash
 # Inside <path to onemkl>
 md build && cd build
-cmake .. -G Ninja
-                  [-DMKL_ROOT=<mkl_install_prefix>] \                    # required only if environment variable MKLROOT is not set
-                  [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \    # required only for testing
-                  [-DREF_LAPACK_ROOT=<reference_lapack_install_prefix>]  # required only for testing
-
+cmake .. -G Ninja [-DCMAKE_CXX_COMPILER=<path_to_dpcpp_compiler>\bin\dpcpp] \  # required only if dpcpp is not found in environment variable PATH
```

Comment:
Following the Linux approach and CMake recommendations, can we add `set CXX=<path_to_dpcpp_compiler>/bin/dpcpp` before the `cmake` command?